com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/antlr4/com/ca/lsp/core/cobol/parser/CobolPreprocessor.g4

```diff
@@ -88,156 +88,270 @@ statement
    ;
 
 moveStatement
-   : MOVE ~(IF | DOT_FS | GO)*?
-   ;
+   : MOVE ~(ACCEPT | ADD | ALTER | CALL | CANCEL | CLOSE | COMPUTE | CONTINUE | DELETE | DISABLE | DISPLAY | DIVIDE | ENABLE |
```

Comment:
Is there any way to incapsulate the duplicating code?

Reply:
it is not duplicate if you are watching closely, every time one keyword is changing but I can try to investigate on that